ROADMAP.md

```diff
@@ -1,40 +1,67 @@
-  * Mimic support
-  * Improved data placement and pool configuration (CRUSH maps)
-  * Dynamic Volume Provisioning for CephFS
-  * ceph-volume provisioning for OSDs
-  * Ceph CSI plug-in that runs everywhere (rook-ceph-agent uses ceph-fuse, nbd-rbd / tcmu runner)
-* Cluster and Pool types are declared Beta
+* Update project governance policies [#1445](https://github.com/rook/rook/issues/1445)
+* Add Core Infrastructure Initiative (CII) Best Practices [#1440](https://github.com/rook/rook/issues/1440)
+* Integrate a more robust controller framework (e.g., CoreOS Operator SDK or Kubebuilder) [#1981](https://github.com/rook/rook/issues/1981)
+* Build and integration testing improvements
+  * Increase PR quality gates (e.g., [vendoring verification](https://github.com/rook/rook/issues/1822), [license scanning](https://github.com/rook/rook/issues/1993), etc.)
+  * Update promotion and release channels to align with storage provider specific statuses [#1885](https://github.com/rook/rook/issues/1885)
+  * Refactor test framework and helpers to support multiple storage providers [#1788](https://github.com/rook/rook/issues/1788)
+  * Isolate and parallelize storage provider testing [#1218](https://github.com/rook/rook/issues/1218)
+  * Longhaul testing pipeline [#1847](https://github.com/rook/rook/issues/1847)
+* Custom resource validation, progress, status [#1539](https://github.com/rook/rook/issues/1539)
+* Design for Volume Snapshotting and policies (consider aligning with SIG-storage) [#1552](https://github.com/rook/rook/issues/1552)
+* Support for dynamic provisioning of new storage types
+  * Dynamic bucket provisioning [#1705](https://github.com/rook/rook/issues/1705)
+  * Dynamic database provisioning [1704](https://github.com/rook/rook/issues/1704)
+* New storage providers
+  * NFS operator and CRDs (backed by arbitrary PVs) [#1551](https://github.com/rook/rook/issues/1551)
+  * Cassandra design [#1910](https://github.com/rook/rook/issues/1910)
+* CockroachDB
+  * Secure deployment using certificates [#1809](https://github.com/rook/rook/issues/1809)
+  * Helm chart deployment [#1810](https://github.com/rook/rook/issues/1810)
+  * Run on arbitrary PVs [#919](https://github.com/rook/rook/issues/919)
+* Minio
+  * Helm chart deployment [#1814](https://github.com/rook/rook/issues/1814)
+  * Run on arbitrary PVs [#919](https://github.com/rook/rook/issues/919)
+* Ceph
+  * Update to Mimic [#1536](https://github.com/rook/rook/issues/1536)
+  * Automated upgrade support (initial) [#997](https://github.com/rook/rook/issues/997)
+  * Manage an existing Ceph cluster (basic) [#1868](https://github.com/rook/rook/issues/1868)
+  * OSDs
+    * Run on arbitrary PVs (local storage) as an alternative to host path [#796](https://github.com/rook/rook/issues/796) [#919](https://github.com/rook/rook/issues/919)
+    * Minimize or eliminate running with privileged containers (e.g., reduced access to /dev) [#1944](https://github.com/rook/rook/issues/1944)
```

Comment:
There is a comma too much after the `e.g.`

Reply:
I prefer the style of including a comma after i.e. and e.g..  From https://www.dailywritingtips.com/comma-after-i-e-and-e-g/:
> The Chicago Manual of Style states that i.e. and e.g. should be “confined to parentheses and notes and followed by a comma.”

> The AP Stylebook, whose “punctuation-pitch” leans generally to the side of “the fewer commas the better,” is pro-comma when it comes to i.e. and e.g. According to AP, both abbreviations are “always followed by a comma.”